docs/howtos/releases.md

```diff
@@ -145,8 +145,9 @@ If you want to uplift changes into a previous release:
 
 * Make sure the changes are present in `main` and have been thoroughly tested
 * Checkout the `release-vXXX` branch, where `XXX` is the major version number
-* Create a PR to bump the version in `version.txt` from [release_version].0 to [release_version].0.1 on the release branch
+  * Do not bump `version.txt`, this is handled by mergify.
 * Cherry-pick any commits that you want to uplift into a PR or ensure all the needed PRs are merged into the release branch
+* Create a PR to bump the backport the changes to the `release-vXXX` branch.
```

Comment:
IMO we should update these docs to tell people to use the `@mergify backport release-vXXX` command instead of all the manual steps currently being called out.

Reply:
Agreed.  I updated the docs to say this instead.  Tell me if I messed up any of the new details.